RecoMTD/TransientTrackingRecHit/plugins/MTDTransientTrackingRecHitBuilderESProducer.h

```diff
@@ -0,0 +1,37 @@
+#ifndef RecoMTD_TransientTrackingRecHit_MTDTransientTrackingRecHitBuilderESProducer_H
+#define RecoMTD_TransientTrackingRecHit_MTDTransientTrackingRecHitBuilderESProducer_H
+
```

Comment:
contents of this file can be merged into MTDTransientTrackingRecHitBuilderESProducer.cc